From 8280fd7ec6cff6db6463c8a1b01d2e427e418226 Mon Sep 17 00:00:00 2001
From: Srinivas Pandruvada <srinivas.pandruvada@linux.intel.com>
Date: Tue, 3 May 2016 13:15:44 -0700
Subject: [PATCH] Downgrade error

When there are no relationship tables present, we print error.
This causes confusion. So downgrade error as thermald will continue
to use its defaults.
---
 src/thd_trt_art_reader.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/thd_trt_art_reader.cpp b/src/thd_trt_art_reader.cpp
index 727e36e..20cf036 100644
--- a/src/thd_trt_art_reader.cpp
+++ b/src/thd_trt_art_reader.cpp
@@ -30,7 +30,7 @@
 
 using namespace std;
 
-#define PRINT_ERROR(...)	thd_log_error(__VA_ARGS__)
+#define PRINT_ERROR(...)	thd_log_info(__VA_ARGS__)
 #define PRINT_DEBUG(...)	thd_log_debug(__VA_ARGS__)
 
 typedef struct {
-- 
2.8.1

